src/Network/Response.php

```diff
@@ -546,10 +546,10 @@ protected function _sendHeader($name, $value = null)
     protected function _sendContent($content)
     {
         if (!is_string($content) && is_callable($content)) {
-            call_user_func($content);
-        } else {
-            echo $content;
+            $content = $content();
         }
+
+        echo $content;
```

Comment:
This could result in additional `1` being output if the callable returns `true`.